cli/admin/src/main/java/io/pravega/cli/admin/utils/TableSegmentUtils.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.utils;
+
+
+import com.google.common.base.Preconditions;
+import io.pravega.client.tables.impl.TableSegmentEntry;
+import io.pravega.common.util.ByteArraySegment;
+import io.pravega.segmentstore.server.tables.EntrySerializer;
+import lombok.Getter;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TableSegmentUtils {
+
+    /**
+     * Retrieve Pravega Operations from chunk files.
+     * @param chunkFiles Chunks list of chunk files to be parsed.
+     * @return List of Pravega operations.
+     * @throws IOException if any exception while parsing raw chunk files.
+     */
+    public static List<TableSegmentUtils.TableSegmentOperation>  getOperationsFromChunks(List<File> chunkFiles) throws IOException {
+        byte[] partialEntryFromLastChunk = null;
+        int unprocessedBytesFromLastChunk = 0;
+        List<TableSegmentUtils.TableSegmentOperation> tableSegmentOperations = new ArrayList<>();
+        for (File f : chunkFiles) {
+            Preconditions.checkState((unprocessedBytesFromLastChunk > 0 && partialEntryFromLastChunk != null) || (unprocessedBytesFromLastChunk == 0 && partialEntryFromLastChunk == null));
```

Comment:
Can we add some extra checks here to always make sure we are somehow not reading files out of sequence?

Reply:
The caller of this method ensures the correct order of files are passed. Added a NOTE explaining the onus lying on the caller of the method.